src/types/app-manager.ts

```diff
@@ -434,7 +434,7 @@ export class AppManager {
 
         // If this is a number, first replace any byte representations of the number
         // These may appear in the TEAL in order to circumvent int compression and preserve PC values
-        if (typeof value === 'number' || typeof value === 'boolean') {
```

Comment:
Boolean still makes sense (converts to `0`/`1`)

Reply:
Thoughts @joe-p?

Reply:
I suppose it depends on if we expect booleans to be ABI encoded or not. How does Puya currently handle boolean template variables?

Reply:
If we want ABI encoded the current implementation would work if you passed through a uint8array rather than a boolean. TBF maybe we should only allow bigint and uint8array at this low level anyway, and then it's the higher level app client construct that does the fancy stuff, that matches the way I've implemented the rest of app client stuff vs composer.

Reply:
I just realised we don't even allow boolean anyway.